engineering/ember.md

```diff
@@ -1 +1,11 @@
 # Ember Style Guide
+
+## Test
+
+### Test selector
+
+We try to use page objects as much as possible, but if we need to use test-selector directly in the test, import it using the following syntax:
```

Comment:
You are right. I think I want to make clear that we should hide as much as possible in the page objects and then if we need to use test selector directly, then let's import with the syntax mentioned.

But I think you are right, maybe that's  a different chapter, let's keep only to document the syntax that we'll be using.